src/test.js

```diff
@@ -64,6 +64,10 @@ export default function Test( settings ) {
 		this.async = false;
 		this.expected = 0;
 	} else {
+		this.module.unskippedTests.push( {
```

Comment:
Blerg. Turns out we need to keep an unskipped count. I was hoping to just look at the list of tests and when we run the first/last unskipped test, run the before/after hook, but when looking through the `module`/`childModule`/`tests` tree, we can't tell the difference between this:

```
QUnit.module("outer", function() {
  test("outer test", function() { ... });
  QUnit.module("inner", function() {
    test("inner test", function() { ... });
  });
});
```

and this:

```
QUnit.module("outer", function() {
  QUnit.module("inner", function() {
    test("inner test", function() { ... });
  });
  test("outer test", function() { ... });
});
```

So we don't know whether to run `outer`'s `before` hook when we run `outer` or `inner`. Without some knowledge of how a module's tests are ordered with respect to its child modules, this approach will not work.

So I guess I'll fall back on still adding the `skip` flag to `module.tests` (so we can dynamically count the number of unskipped tests in a module and don't have to keep a separate list/count of them), but also keeping an `unskippedTestsRun` on the module.